js/authentication.md

```diff
@@ -845,7 +918,7 @@ const oauth = {
   scope: ['phone', 'email', 'profile', 'openid', 'aws.cognito.signin.user.admin'],
   redirectSignIn: 'http://localhost:3000/',
   redirectSignOut: 'http://localhost:3000/',
-  responseType: 'code' // or token
+  responseType: 'code' // or 'token', note that REFRESH token will only generated when the responseType is code
```

Comment:
Missing `be` in  `note that REFRESH token will only generated when the responseType is code` should be place before the `generated`.

Should read: note that REFRESH token will only **be** generated when the responseType is code